pandas/core/indexes/frozen.py

```diff
@@ -51,6 +51,9 @@ def __eq__(self, other):
             other = list(other)
         return super(FrozenList, self).__eq__(other)
 
+    def __ne__(self, other):
```

Comment:
It looks like this inherits from `list`, which presumably takes care of `__ne__`?